runtime/tests/mock.rs

```diff
@@ -143,6 +143,7 @@ impl ExtBuilder {
 		crml_staking::GenesisConfig::<Runtime> {
 			minimum_bond: 1,
 			current_era: 0,
+			minimum_bond: 1,
```

Comment:
This line is repeated

Reply:
Ah I fixed, then rebased so that must be where this came from